src/symbolicEngine/symbolicEngine.cpp

```diff
@@ -310,10 +347,15 @@ SymbolicVariable *SymbolicEngine::convertRegToSymVar(uint64 regId, uint64 symVar
     throw std::runtime_error("SymbolicEngine::convertRegToSymVar() - Invalid register ID");
 
   regSymId = this->getRegSymbolicID(regId);
-  if (regSymId == UNSET)
-    throw std::runtime_error("SymbolicEngine::convertRegToSymVar() - This register ID is UNSET");
 
-  expression = this->getExpressionFromId(regSymId);
+  if (regSymId == UNSET){
+    std::stringstream     expr("#0");
```

Comment:
newSymbolicExpression(std::stringstream &src) takes a source id in parameter but as we create a new symbolic variable we don't have source id so I took the #0 value.